_schemaV2.graphql

```diff
@@ -23208,12 +23211,6 @@ type ShippingPreset implements Node {
 
   # Whether Artsy handles international shipping
   artsyShippingInternational: Boolean
-  createdAt(
```

Comment:
removing `createdAt` and `updatedAt` from `ShippingPresetType`

They are not used anywhere plus they were using a deprecated type